### DISCUSSION TITLE
Vertical text selection

### DISCUSSION THREAD

**Discussion by HubKing** [tsujan/FeatherPad#789] [Ideas]
VS Code has it (middle mouse dragging), but VS Code is too heavy. FeatherPad does not seem to have this feature. If it would not be a difficult task, how about adding vertical selection? It can be useful when trying to remove unnecessary parts from logs or programmatic output.

![image](https://github.com/tsujan/FeatherPad/assets/10900489/80cf941b-097f-4dc8-9f54-c792dd0f94e9)

**Comment by tsujan** [comment]
Since the selection is done by Qt and not FeatherPad, I don't think it's possible without adding a very large amount of code. That being said, I'll look into it. If the needed code isn't as large as I imagine, why not? Thanks for the suggestion!

**Comment by tsujan** [comment]
See https://github.com/tsujan/FeatherPad/issues/801#issuecomment-2561962237

EDIT: Done.
![column](https://github.com/user-attachments/assets/a6e37643-97e7-45df-b3f7-82a6fef76e5a)
